code/modules/uplink/uplink_item.dm

```diff
@@ -558,6 +559,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/mecha/combat/gygax/dark/loaded
 	cost = 80
 
+/datum/uplink_item/dangerous/guardian
+	name = "Holoparasites"
+	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an organic host as a home base and source of fuel."
+	item = /obj/item/weapon/storage/box/syndie_kit/guardian
+	cost = 20
+
```

Comment:
That's weird did you copy paste it or write it out? Probably better to copy it from another line so line 2229 where the surplus crate excludes from other modes than traitor you can just copy that line and paste it at line 567.